src/html/js/common_personium.js

```diff
@@ -298,7 +298,7 @@ Common.appendRequestCells = function(extUrl, dispName) {
     $("#bSendAllowed").prop("disabled", false);
 };
 
-Common.sendMessageAPI = function(uuid, extCell, type, title, body, reqRel, reqRelTar) {
+Common.sendMessageAPI = function(uuid, extCell, type, title, body, reqType, reqRel, reqRelTar) {
```

Comment:
Please add the following comment to explain why suddenly we don't need to specify App URL anymore.  

    /*
     * When the following conditions are satisfied, there is no need to include App URL when specifying the role/relation name.
     * 1. BoxBound must set to true
     * 2. Authorization token must be App authenticated token
     */